intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -592,8 +671,9 @@ def validate_ip(value):
         ('source.geolocation.region', 'region'),
         ('source.geolocation.city', 'city'),
     ],
-    'classification_type': 'vulnerable service',
-    'feed_url': 'https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-Memcached'
```

Comment:
* Because `feed.url` defines the source of the information, if downloaded. Not the documentation/specification.
* This field is added by the collector, not by a parser.